administrator/components/com_contact/helpers/associations.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_associations
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Contact Associations helper.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class ContactAssociationsHelper extends JHelperContent
+{
+	/**
+	 * Check if item supports associations
+	 *
+	 * @return  boolean
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static function hasAssociationsSupport()
+	{
+		return true;
+	}
+
+	/**
+	 * Check if item supports category associations
+	 *
+	 * @return  boolean
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static function hasAssociationsCategories()
+	{
+		return true;
+	}
+
+	/**
+	 * Mapping of database columns alias.
+	 *
+	 * @return  array
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static function getListQuery($itemType = '')
+	{
+		// Create a new query object.
+		$db = $this->getDbo();
```

Comment:
When using a public static method, I guess you should use `$db = JFactory::getDbo();` and not $this.
Same elsewhere.